template.env

```diff
@@ -97,6 +97,11 @@ SYNCSERVERS_1_KEY=
 # 	SYNCSERVERS_1_PULL_RULES='{\"tags\":{\"OR\":[],\"NOT\":[]},\"orgs\":{\"OR\":[],\"NOT\":[]},\"url_params\":\"{\\\"searchanalysis\\\": \\\"2\\\"}\"}'
 SYNCSERVERS_1_PULL_RULES=
 
+# Supervisord host for external jobs
```

Comment:
external jobs?

Reply:
prepares the ground for running jobs in another container

Reply:
edited the template comment a bit to clarify